tests/nemo_text_processing/es/data_text_normalization/test_cases_whitelist.txt

```diff
@@ -2,6 +2,5 @@ el dr.~el doctor
 sr. rodriguez~señor rodriguez
 182 esq. toledo~ciento ochenta y dos esquina toledo
 da. ana pérez~doña ana pérez
-ee. uu.~estados unidos
```

Comment:
Why remove?

Reply:
I think it was due to failing tests. I readded, retested, and all is fixed.